django-rest/imdb_app/imdb_api/views.py

```diff
@@ -31,4 +45,16 @@ class CommentViewSet(viewsets.ModelViewSet):
     permission_classes = [permissions.IsAuthenticated]
     filter_backends = [DjangoFilterBackend]
     filterset_fields = ['movie']
+
+class LikeViewSet(mixins.CreateModelMixin, 
```

Comment:
Ista prica kao i pre, niko ne sme da obrise like od jedne osobe samo ta osoba sme da obrise, ovde po ovome moze da obrise bilo ko ako je autentifikovan i posalje id

Reply:
Kako isto hendlas to da user moze da lajkuje za samog sebe? Da ne mogu da posaljem tudj id i like-ujem za drugu osobu, treba da uzima aktivnog usera na serializeru to moze kao field da se podesi ali nisam siguran da li sam video to